qa-tests-backend/src/main/groovy/services/SignatureIntegrationService.groovy

```diff
@@ -0,0 +1,75 @@
+package services
+
+import io.grpc.Status
+import io.grpc.StatusRuntimeException
+import io.stackrox.proto.api.v1.EmptyOuterClass
+import io.stackrox.proto.api.v1.SignatureIntegrationServiceGrpc
+import io.stackrox.proto.api.v1.SignatureIntegrationServiceOuterClass
+import io.stackrox.proto.storage.ImageIntegrationOuterClass
+import io.stackrox.proto.storage.SignatureIntegrationOuterClass
+import util.Timer
+
+class SignatureIntegrationService extends BaseService {
```

Comment:
```suggestion
@CompileStatic
class SignatureIntegrationService extends BaseService {
```
> This will let the Groovy compiler use compile time checks in the style of Java then perform static compilation, thus bypassing the Groovy meta object protocol.
>
> When a class is annotated, all methods, properties, files, inner classes, etc. of the annotated class will be type checked. When a method is annotated, static compilation applies only to items (closures and anonymous inner classes) within the method.
http://docs.groovy-lang.org/next/html/gapi/groovy/transform/CompileStatic.html